### PR TITLE
Mace pretrained

### PR DESCRIPTION
**Pull request by sblackburn86**
this PR allows us to download & load a pre-trained MACE in our architecture. It works with CUDA.

I had to change the z_table used in the construction of the graphs and MACE to have 89 atom types, this is necessary for the pre-trained models. It will also help us in the future when we deal with more than 1 atom type. I used a list(range(89)) which implies a type Z=0. Not sure what that would be. Maybe a masked atom type?

### CODE REVIEW

**Review by rousseab — APPROVED** [review]
LGTM!
